auto_cache_examples/example_wf.py

```diff
@@ -0,0 +1,29 @@
+# import numpy as np
+# import pandas as pd
+
+
+from flytekit import task, workflow
+from flytekit.auto_cache.auto_cache import AutoCacheConfig
+
+
+@task(auto_cache_config=AutoCacheConfig(check_task=True, check_modules=True, check_packages=True))
```

Comment:
I'd add a default AutoCacheConfig to the task and set all the parameters to true, so users don't need to specify it by default. However, I'm not sure if it's too aggressive.